src/Microsoft.AspNet.Mvc.ModelBinding/ModelMetadata.cs

```diff
@@ -197,6 +207,10 @@ public string PropertyName
         /// Gets a value indicating whether or not the model value is required. This is only applicable when
         /// the current instance represents a property.
         /// </summary>
+        /// <remarks>
+        /// If <c>true</c> then the model value is considered required by validators and must have a non-<c>null</c>
+        /// value to be considered valid.
```

Comment:
I'm going to change this comment then to mention that we're using an implicit `[Required]`